cpp/include/cudf/stream_compaction.hpp

```diff
@@ -254,20 +251,19 @@ std::unique_ptr<table> unique(
 /**
  * @brief Create a new table without duplicate rows.
  *
- * Given an `input` table_view, each row is copied to output table if the corresponding
+ * Given an `input` table_view, each row is copied to the output table if the corresponding
  * row of `keys` columns is distinct (no other equivalent row exists in the table). If duplicate
- * rows are present, it is unspecified which row is copied.
+ * rows are present, depending on the value of `keep`:
+ * - KEEP_FIRST: only the first of a sequence of duplicate rows is copied
+ * - KEEP_LAST: only the last of a sequence of duplicate rows is copied
+ * - KEEP_ANY: an unspecified row in a sequence of duplicate rows is copied
+ * - KEEP_NONE: no duplicate rows are copied
```

Comment:
This seems like redundant with the documentation of `duplicate_keep_option`.

Reply:
Right. This seems redundant because `duplicate_keep_option` enum already has docs about these behaviors. Let drop them.